packages/aws-cdk-lib/aws-apigatewayv2/lib/websocket/integration.ts

```diff
@@ -267,6 +310,14 @@ export interface WebSocketRouteIntegrationConfig {
    */
   readonly templateSelectionExpression?: string;
 
+  /**
+   * The maximum amount of time an integration will run before it returns without a response.
+   * Must be between 50 milliseconds and 29 seconds.
+   *
+   * @default Duration.seconds(29)
```

Comment:
nit: can you add a hyphen to match the other defaults ?  😄

Reply:
I think the convention is supposed to be `@default {defaultValue} - {description}`. This is one is `@default {defaultValue}`, and the others are just `@default - {description}`

Reply:
ah makes sense. thanks!